LOCALIZE.md

```diff
@@ -1,67 +1,57 @@
-# Localization steps for KitkitSchool Applications
+## Steps to localize only instructions
+1. Add a new `productFlavor` in the [build.gradle](https://github.com/maqsoftware/Pehla-School/blob/newmaster/pehlaschool/proj.android-studio/app/build.gradle) file along with `applicationSuffixId` property in it.
+2. Add the new language in the `localeType` enum in LanguageManager.hpp file.
+3. Add a case statement in the `getLocalizedString()` method for a new language. Also, add a case statement for the new language in the files viz. `CompletePopup.cpp, TodoLoadingScene.cpp, NumberMatchingScene.cpp, MultipleChoicesScene.cpp, LetterMatchingScene.cpp, FillTheBlanksScene.cpp, LetterTraceMainDepot.cpp, ReorderingScene.cpp, CompTraceScene.cpp, MatchingScene.cpp, LRDividedTypeReorderingScene.cpp, LRDividedTypeFillTheBlanksScene.cpp, LRAllInOneTypeQuestionScene.cpp, LRDividedTypeQuestionScene.cpp, LRComprehensionScene.cpp, WordTraceMainDepot.cpp, AnswerPadMulti.cpp, AnswerPadSingle.cpp, GradeSelector.cpp.`
+4. Localize the [game tutorial](https://github.com/maqsoftware/Pehla-School-Assets/tree/master/localized/tutorialvideo) videos.
+4. [curriculumdata_levels_en.tsv](https://github.com/maqsoftware/Pehla-School-Assets/blob/master/localized/curriculumdata_en.tsv) file contains mapping of game names for different days. It contains titles like "PreSchool", "English 1" or "Math 3" that can be localized.
+5. [eggquizliteracy_levels_en.tsv](https://github.com/maqsoftware/Pehla-School-Assets/blob/master/localized/games/eggquiz/eggquizliteracy_levels_en.tsv) and [eggquizmath_levels_en.tsv](https://github.com/maqsoftware/Pehla-School-Assets/blob/master/localized/games/eggquiz/eggquizmath_levels_en.tsv) files contains instructions for the questions to be answered in the quiz sections that can be localized.
+6. [wordwindow_levels_en.tsv](https://github.com/maqsoftware/Pehla-School-Assets/blob/master/localized/games/wordwindow/wordwindow_level_en.tsv) contains word problems for doing math that can be localized.
+7. Add locale specific splash screen and falling Pehla School logo images in [localized->system](https://github.com/maqsoftware/Pehla-School-Assets/tree/master/localized/system) and [main->system](https://github.com/maqsoftware/Pehla-School-Assets/tree/master/main/system) folders respectively.
+8. Localize all the resources in [res](https://github.com/maqsoftware/Pehla-School/tree/newmaster/pehlaschool/proj.android-studio/app/src/english/res) folder and create a new folder the new language.
+9. Add a new translation string to the vector in the CPP map initialization and a condition for the new language below it in the `DailyScene2`.cpp file.
+10. Update the TTS locale settings according to the new language in VoiceMoldManager.java.
```

Comment:
Add a step to create language-specific AndroidManifest for Content Provider changes